packages/shared_preferences/shared_preferences/pubspec.yaml

```diff
@@ -28,6 +28,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
+  leak_tracker_flutter_testing: any
```

Comment:
This is in the wrong dependency section.

https://github.com/flutter/packages/pull/8365 will enforce this.

Reply:
Ooops sorry about that! I was talking to @polina-c and she was suggesting to not activate leak tracking for packages with no disposable classes for now. Hence, I'll close this PR